docs/exchanges/okx.md

```diff
@@ -7,16 +7,20 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/okx)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://www.okx.com/join/1931920>
+* Sign up with OKEX using Hummingbot's referral link for a 10% discount!
```

Comment:
* Sign up with OKX using Hummingbot's referral link for a 10% discount off trading fees!